django/template/engine.py

```diff
@@ -29,7 +31,13 @@ def __init__(
         libraries=None,
         builtins=None,
         autoescape=True,
+        multiline=False,
```

Comment:
```suggestion
        multiline=False,  # RemovedInDjango70Warning: When the deprecation ends, replace with: multiline=True,
```
We need comments to explain the required updates after the deprecation, perhaps something like the above